### PR TITLE
Adds tab indicator to the build page

### PR DESCRIPTION
**Pull request by enejb**
This updates the favicon so that it includes a dot when building telling you the progress of the build.

Before:
screen shot 2018-11-26 at 10 05 28 am

After: 
screen shot 2018-11-26 at 10 05 00 am

Notice the dot. 
I wasn't able to fully test the change.

### CODE REVIEW

**Comment by brbrr** [comment]
@oskosk what do you think, should we try this out?

File: jurassicninja.js [hunk 1]
```diff
@@ -185,3 +188,81 @@ function jurassicNinjaApi() {
...
+ */
+function favicon_update_colour( colour ) {
+
+    function getLocation( href ) {
+        var l = document.createElement( 'a' );
+        l.href = href;
+        return l;
+    }
+
+		function remove_all_favicons() {
```
Comment (brbrr) [inline jurassicninja.js]: Some indentation issues in this function. spaces vs tabs?

**Comment by enejb** [comment]
Updated with the fixes.

**Comment by oskosk** [comment]
Looks awesome. Merging!

**Comment by oskosk** [comment]
Well I merged this one cause I saw it working on the staging server, but it does not work on production. Maybe a different favicon format is the cause ?

**Pull request merged**